.storybook/webpack.config.js

```diff
@@ -73,5 +73,19 @@ module.exports = webpackSettings => {
     })
   );
 
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    use: [
+      {
+        loader: require.resolve('awesome-typescript-loader'),
```

Comment:
Did you look into using `babel-loader` here, too? https://storybook.js.org/docs/configurations/typescript-config/#setting-up-typescript-with-babel-loader

Reply:
Replaced to babel-loader, removed the previous one.